scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -1009,8 +1029,14 @@ def _benchmark_cnn(self):
       variable_mgr_init_ops.extend(self.variable_mgr.get_post_init_ops())
     local_var_init_op_group = tf.group(*variable_mgr_init_ops)
 
-    summary_op = tf.summary.merge_all()
     is_chief = (not self.job_name or self.task_index == 0)
+    if self.params.variable_update == 'horovod':
+      import horovod.tensorflow as hvd
+      if hvd.rank() > 0:
+        # Only first worker can be chief.
```

Comment:
do we also need to set it to true when hvd.rank() is 0?  (that is, when will hvd.rank() differ from self.task_index ?)